tests/unit/models/governing-body-test.js

```diff
@@ -0,0 +1,828 @@
+
+      const isValid = await model.validate();
+
+      assert.true(isValid);
+      assert.notOk(model.error, 'model.error should not be undefined');
+
+      otherBodiesStub.restore();
+    });
+
+    test("it returns no errors when the model's start date is strictly within the period of another governing body and they have the same end date", async function (assert) {
+      const otherModel = this.store().createRecord('governing-body', {
+        startDate: new Date('2023-01-01'),
+        endDate: new Date('2024-01-01'),
+      });
+
+      const model = this.store().createRecord('governing-body', {
+        startDate: new Date('2023-06-06'),
+        endDate: new Date('2024-01-01'),
+      });
+
+      const otherBodiesStub = sinon.stub(model, 'getOtherTimedGoverningBodies');
+      otherBodiesStub.resolves([otherModel]);
+
+      const isValid = await model.validate();
+
+      assert.true(isValid);
+      assert.notOk(model.error, 'model.error should not be undefined');
+
+      otherBodiesStub.restore();
+    });
+
+    test("it returns no errors when the model's end date is strictly within the period of another governing body and they have the same start date", async function (assert) {
+      const otherModel = this.store().createRecord('governing-body', {
+        startDate: new Date('2023-01-01'),
+        endDate: new Date('2024-01-01'),
+      });
+
+      const model = this.store().createRecord('governing-body', {
+        startDate: new Date('2023-01-01'),
+        endDate: new Date('2023-06-06'),
+      });
+
+      const otherBodiesStub = sinon.stub(model, 'getOtherTimedGoverningBodies');
+      otherBodiesStub.resolves([otherModel]);
+
+      const isValid = await model.validate();
+
+      assert.true(isValid);
+      assert.notOk(model.error, 'model.error should not be undefined');
+
+      otherBodiesStub.restore();
+    });
+
+    test('it should not check start date overlap unless it was changed', async function (assert) {
+      const model = this.store().createRecord('governing-body', {
+        startDate: new Date('2024-02-01'),
+        endDate: new Date('2026-01-01'),
+      });
+
+      const changedStub = sinon.stub(model, 'changedAttributes');
+      changedStub.returns({});
+
+      const isValid = await model.validate();
+
+      assert.true(isValid);
+      assert.notOk(model.error, 'model.error should be undefined');
+
+      changedStub.restore();
+    });
+
+    test('it should not check end date overlap unless it was changed', async function (assert) {
+      const model = this.store().createRecord('governing-body', {
+        startDate: new Date('2023-01-01'),
+        endDate: new Date('2024-02-02'),
+      });
+
+      const changedStub = sinon.stub(model, 'changedAttributes');
+      changedStub.returns({});
+
+      const isValid = await model.validate();
+
+      assert.true(isValid);
+      assert.notOk(model.error, 'model.error should be undefined');
+
+      changedStub.restore();
+    });
```

Comment:
These tests are similar, only one can be keep and rename is something more generic like: "it should not check date overlap unless it was changed"